hypershift-operator/controllers/sharedingress/router.go

```diff
@@ -88,10 +92,11 @@ func generateRouterConfig(svcList *corev1.ServiceList, svcsNamespaceToClusterID
 	p.Backends = make([]backendDesc, 0, len(svcList.Items))
 	for _, svc := range svcList.Items {
 		p.Backends = append(p.Backends, backendDesc{
-			Name:      svc.Namespace + "-" + svc.Name,
-			SVCIP:     svc.Spec.ClusterIP,
-			SVCPort:   svc.Spec.Ports[0].Port,
-			ClusterID: svcsNamespaceToClusterID[svc.Namespace],
+			Name:         svc.Namespace + "-" + svc.Name,
+			SVCIP:        svc.Spec.ClusterIP,
+			SVCPort:      svc.Spec.Ports[0].Port,
+			ClusterID:    svcsNamespaceToClusterID[svc.Namespace],
+			AllowedCIDRs: setAllowedCIDRs(svcsNamespaceToApiServerNetworking[svc.Namespace]),
```

Comment:
Why set this at all if there isn't a CIDR configued?

Reply:
@jharrington22 Thanks for the review!
Done. Note that I've done this conditional formatting **inside the template** otherwise there would be redundant lines in the template.
In this manor- If no `AllowedCIDRBlocks` specified [or they where removed by the client] the template is exactly the same as it was previously. 
Without these conditionals we would likely need to write `0.0.0.0/0` to specify explicitly as in previous commit. 
I hope that makes sense.